spec/libsass-todo-issues/issue_947/expected.compact.css

```diff
@@ -0,0 +1,2 @@
+@keyframes test { 10% { color: red; } }
+
```

Comment:
Yup we have `.trim()` in node-sass spec runner too. Would be nice if we enforce one rule for all specs. :sunglasses: